app/test/prepare_proposal_context_test.go

```diff
@@ -33,40 +32,41 @@ func TestTimeInPrepareProposalContext(t *testing.T) {
 	vestAccName := "vesting"
 	cfg := testnode.DefaultConfig().WithFundedAccounts(sendAccName)
 	cctx, _, _ := testnode.NewNetwork(t, cfg)
-	ecfg := encoding.MakeConfig(app.ModuleEncodingRegisters...)
+	ecfg := encoding.MakeConfig()
 
 	type test struct {
 		name    string
 		msgFunc func() (msgs []sdk.Msg, signer string)
 	}
 	tests := []test{
-		{
-			name: "create continuous vesting account with a start time in the future",
-			msgFunc: func() (msgs []sdk.Msg, signer string) {
-				_, _, err := cctx.Keyring.NewMnemonic(vestAccName, keyring.English, "", "", hd.Secp256k1)
-				require.NoError(t, err)
-				sendingAccAddr := testfactory.GetAddress(cctx.Keyring, sendAccName)
-				vestAccAddr := testfactory.GetAddress(cctx.Keyring, vestAccName)
-				msg := vestingtypes.NewMsgCreateVestingAccount(
-					sendingAccAddr,
-					vestAccAddr,
-					sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1000000))),
-					time.Now().Unix(),
-					time.Now().Add(time.Second*100).Unix(),
-					false,
-				)
-				return []sdk.Msg{msg}, sendAccName
-			},
-		},
+		// TODO: how should this test function in 0.52?
+		// {
+		// 	name: "create continuous vesting account with a start time in the future",
+		// 	msgFunc: func() (msgs []sdk.Msg, signer string) {
+		// 		_, _, err := cctx.Keyring.NewMnemonic(vestAccName, keyring.English, "", "", hd.Secp256k1)
+		// 		require.NoError(t, err)
+		// 		sendingAccAddr := testfactory.GetAddress(cctx.Keyring, sendAccName)
+		// 		vestAccAddr := testfactory.GetAddress(cctx.Keyring, vestAccName)
+		// 		msg := vestingtypes.NewMsgCreateVestingAccount(
+		// 			sendingAccAddr,
+		// 			vestAccAddr,
+		// 			sdk.NewCoins(sdk.NewCoin(app.BondDenom, math.NewInt(1000000))),
+		// 			time.Now().Unix(),
+		// 			time.Now().Add(time.Second*100).Unix(),
+		// 			false,
+		// 		)
+		// 		return []sdk.Msg{msg}, sendAccName
+		// 	},
+		// },
```

Comment:
[question]

is it possible to keep the existing vesting module or maintain all of its functionality? all of the legal / ops is built around it